### PR TITLE
add login validator

### PR DESCRIPTION
**Pull request by patrykbrodziak1**

### CODE REVIEW

**Review by Burbinox — APPROVED** [review]
LGTM

File: common/validators.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
+import * as yup from "yup";
+
+const loginSchema = yup.object({
+  body: yup.object({
+    username: yup.string().required(),
```
Comment (kubaczerwinski77) [inline common/validators.js]: I would suggest signing in using email rather than username.

File: common/validators.js [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+    password: yup.string().required(),
```
Comment (kubaczerwinski77) [inline common/validators.js]: We need more validation rules about password. Think about min / max characters, numbers or special characters.

**Review by kubaczerwinski77 — CHANGES_REQUESTED** [review]
I left some comments about validation rules. Check them out.

File: common/validators.js (hunk 1 above)
Comment (patrykbrodziak1) [reply common/validators.js]: ok, email and password, yes?

Comment (kubaczerwinski77) [reply common/validators.js]: Yup

**Review by kubaczerwinski77 — APPROVED** [review]

**Pull request merged**